addons/packages/metrics-server/0.5.1/bundle/config/schema.yaml

```diff
@@ -0,0 +1,29 @@
+#! schema.yml
+
+#@data/values-schema
+#@schema/desc "OpenAPIv3 Schema for metrics-server"
+---
+#@schema/desc "The namespace in which metrics-server is deployed"
+namespace: kube-system
+metricsServer:
+  #@schema/desc "The namespace in which metrics-server is deployed"
```

Comment:
Probably, we can modify the description of namespace on L11 to be different from L7 namespace description
Refer to something in kapp controller field `kappController.namespace` : "The namespace value used by older templates, will overwrite will top level namespace of present, keep for backward compatibility" (https://github.com/vmware-tanzu/community-edition/blob/main/addons/packages/kapp-controller/0.30.0/README.md)

Reply:
Changed